x/erc20/client/cli/tx.go

```diff
@@ -134,10 +130,6 @@ func NewRegisterERC20ProposalCmd() *cobra.Command {
 				return err
 			}
 
-			if err := msg.ValidateBasic(); err != nil {
-				return err
-			}
-
```

Comment:
There is one more occurrence in this file:
https://github.com/evmos/cosmos-sdk/blob/7a885823bc4a8a72c41dcd1381cba9819f487349/client/tx/tx.go#L35-L43

Could you please also remove that? 😇